src/sagemaker/tensorflow/script_mode_training.rst

```diff
@@ -0,0 +1,146 @@
+Preparing a Script Mode training script
+=======================================
+
+Your TensorFlow training script must be a Python 2.7- or 3.6-compatible source file.
+
+The training script is very similar to a training script you might run outside of SageMaker, but you can access useful properties about the training environment through various environment variables, including the following:
+
+* ``SM_MODEL_DIR``: A string that represents the path where the training job writes the model artifacts to.
```

Comment:
it might be worth calling out that `model_dir` has extra/different meaning for TF